pubsub/google/cloud/pubsub_v1/types.py

```diff
@@ -53,12 +53,13 @@
 # The defaults should be fine for most use cases.
 FlowControl = collections.namedtuple(
     'FlowControl',
-    ['max_bytes', 'max_messages', 'resume_threshold'],
+    ['max_bytes', 'max_messages', 'resume_threshold', 'max_requests'],
```

Comment:
So, to be clear, this will pause the **incoming** stream if the **outgoing** stream becomes congested?

Reply:
right.